src/StoreApi/Routes/CartUpdateCustomer.php

```diff
@@ -119,9 +119,50 @@ protected function get_route_post_response( \WP_REST_Request $request ) {
 		);
 		wc()->customer->save();
 
-		$cart->calculate_shipping();
```

Comment:
`calculate_shipping` was redundant?

Reply:
Cart class has a calculate_totals method which does the same thing.